code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -407,65 +414,197 @@
 
-//Should put a cool menu here, like ninjas.
-/obj/item/clothing/gloves/yautja/hunter/verb/wristblades()
-	set name = "Use Wrist Blades"
-	set desc = "Extend your wrist blades. They cannot be dropped, but can be retracted."
+//bracer attachments
+/obj/item/bracer_attachments
+	name = "wristblade bracer attachment"
+	desc = "Report this if you see this."
+	icon = 'icons/obj/items/hunter/pred_gear.dmi'
+	///Typepath of the weapon attached to the bracer
+	var/obj/item/attached_weapon_type
+	///Reference to the weapon attached to the bracer
+	var/obj/item/attached_weapon
+	///Attachment deployment sound
+	var/deployment_sound
+	///Attachment rectraction sound
+	var/retract_sound
+
+/obj/item/bracer_attachments/Initialize(mapload, ...)
+	. = ..()
+	if(attached_weapon_type)
+		attached_weapon = new attached_weapon_type(src)
+
+/obj/item/bracer_attachments/Destroy()
+	QDEL_NULL(attached_weapon)
+	. = ..()
+
+/obj/item/bracer_attachments/wristblades
+	name = "wristblade bracer attachment"
+	desc = "A pair of huge, serrated blades"
+	icon_state = "wrist"
+	item_state = "wristblade"
+	attached_weapon_type = /obj/item/weapon/bracer_attachment/wristblades
+	deployment_sound = 'sound/weapons/wristblades_on.ogg'
+	retract_sound = 'sound/weapons/wristblades_off.ogg'
+
+/obj/item/bracer_attachments/scimitars
+	name = "scimitar bracer attachment"
+	desc = "A pair of huge, serrated blades"
+	icon_state = "scim"
+	item_state = "scim"
+	attached_weapon_type = /obj/item/weapon/bracer_attachment/scimitar
+	deployment_sound = 'sound/weapons/scims_on.ogg'
+	retract_sound = 'sound/weapons/scims_off.ogg'
+
+/obj/item/bracer_attachments/scimitars_alt
+	name = "scimitar bracer attachment"
+	desc = "A pair of huge, serrated blades"
+	icon_state = "scim_alt"
+	item_state = "scim_alt"
+	attached_weapon_type = /obj/item/weapon/bracer_attachment/scimitar/alt
+	deployment_sound = 'sound/weapons/scims_alt_on.ogg'
+	retract_sound = 'sound/weapons/scims_alt_off.ogg'
+
+/obj/item/clothing/gloves/yautja/hunter/attackby(obj/item/attacking_item, mob/user)
+	if(!istype(attacking_item, /obj/item/bracer_attachments))
+		return ..()
+
+	if(!HAS_TRAIT(user, TRAIT_YAUTJA_TECH))
+		to_chat(user, SPAN_WARNING("You do not know how to attach the [attacking_item] to the [src]."))
+		return
+
+	var/obj/item/bracer_attachments/bracer_attachment = attacking_item
+	if(!bracer_attachment.attached_weapon_type)
+		CRASH("[key_name(user)] attempted to attach the [bracer_attachment] to the [src], with no valid attached_weapon.")
+
+	if(left_bracer_attachment && right_bracer_attachment)
+		to_chat(user, SPAN_WARNING("You already have the maximum amount of bracer attachments on [src]."))
+		return
+
+	var/attach_to_left = TRUE
+	if(!left_bracer_attachment && !right_bracer_attachment)
+		if(tgui_alert(user, "Do you want to attach [bracer_attachment] to the left or right hand?", "[src]", list("Right", "Left",), 15 SECONDS) == "Right") //its right, left because in-game itll show up as left, right
```

Comment:
```suggestion
		if(tgui_alert(user, "Do you want to attach [bracer_attachment] to the left or right hand?", "[src]", list("Right", "Left"), 15 SECONDS) == "Right") //its right, left because in-game itll show up as left, right
```